src/llmcompressor/utils/helpers.py

```diff
@@ -1087,31 +1089,42 @@ def __exit__(self, _exc_type, _exc_val, _exc_tb):
 
 
 @contextlib.contextmanager
-def DisableQuantization(model: torch.nn.Module):
+def DisableQuantization(module: torch.nn.Module):
     """
     Disable quantization from QuantizationModifier
     """
     try:
-        model.apply(disable_quantization)
+        module.apply(disable_quantization)
         yield
     finally:
-        model.apply(enable_quantization)
+        module.apply(enable_quantization)
 
 
 @contextlib.contextmanager
-def calibration_forward_context(model: torch.nn.Module):
+def eval_context(module: torch.nn.Module):
+    restore_value = module.training
+    try:
+        module.train(False)  # equivalent to eval()
```

Comment:
But why not just call `model.eval()` if we're restoring anyway?

Reply:
I pref `.train(False)` just because it's just the most clear and verbose, but as mentioned, it's exactly equivalent to `eval()`